testapp/api.py

```diff
@@ -46,6 +52,17 @@ class Meta:
         fields = ("zaak",)
 
 
+class ZaakObjectFkFilterset(FilterSet):
+    zaak_object = FkOrUrlFieldFilter(
```

Comment:
```suggestion
    zaak = FkOrUrlFieldFilter(
```

It looks like you filter on zaak, so name is a little confusing